packages/utils/src/growth.js

```diff
@@ -0,0 +1,65 @@
+const {
+  GrowthReferral,
+  GrowthInviteCode
+} = require('@origin/growth-event/src/models')
+
+async function _getReferrer(code) {
+  // Lookup the code.
+  const inviteCode = await GrowthInviteCode.findOne({ where: { code } })
+  if (!inviteCode) {
+    throw new Error(`Invalid invite code ${code}`)
+  }
+  return inviteCode.ethAddress
+}
+
+async function makeReferralConnection(code, walletAddress) {
+  const referralLink = await GrowthReferral.findOne({
+    where: {
+      refereeEthAddress: walletAddress.toLowerCase()
```

Comment:
nit: since walletAddress.toLowerCase() is used several times, maybe declare a variable at the top of the method. like
```const referee = walletAddress.toLowerCase()```